comp22/Makefile

```diff
@@ -0,0 +1,11 @@
+
+all: score-sheet-2022.pdf
+
+%.pdf: %.svg
+	inkscape -A colour-$@ $<
+	gs -sOutputFile=$@ -sDEVICE=pdfwrite \
+		-sColorConversionStrategy=Gray -dProcessColorModel=/DeviceGray \
+		-dCompatibilityLevel=1.4 -dNOPAUSE -dBATCH colour-$@
```

Comment:
Is there any reason you aren't using `inkscape --export-pdf=` or `inkscape --export-filename=out.pdf in.svg` in more recent versions.

Reply:
Only that this was copied from previous years'.